services/ai_service.py

```diff
@@ -0,0 +1,23 @@
+from abc import abstractmethod
+
+class AIService:
+    def __init__(self):
+        pass
+
+    def set_image_style(self, image_style):
+        self.image_style = image_style
+
+    # Speak the sentence. Returns None
+    @abstractmethod
+    def run_tts(self, sentence):
```

Comment:
Example of type hinting suggestion if you want to specify a None return (adding potential type hint for the input it will expect as well):

```suggestion
    def run_tts(self, sentence: str) -> None:
```


https://peps.python.org/pep-0484/#using-none